{"name": "test_2_TypeAddClass", "status": "broken", "statusDetails": {"message": "TypeError: 'tuple' object is not callable", "trace": "self = <testcases.test_AddClassification.TestAddClassification object at 0x0491EAF0>\ntest_logsystem = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"26d75486-dab4-469c-b943-611778ebd1b2\")>\n\n    def test_2_TypeAddClass(self,test_logsystem):\n        self.page1 = TestLogPage(test_logsystem)\n        self.page1.type_new_sort_name(\"smart\")\n        self.page1.click_save_sort_name()\n        try:\n            self.page1.click_logclass()\n            time.sleep(2)\n>           assert \"smart\" in self.page1.check_logclass()\nE           TypeError: 'tuple' object is not callable\n\ntest_AddClassification.py:36: TypeError"}, "start": 1628999925526, "stop": 1628999927799, "uuid": "dca215d1-32f7-45b5-a48e-e38cb2bc010d", "historyId": "d242da973f70d4de115e15a504d2bf96", "testCaseId": "34cc12a412b21ac606dd332c398f98e7", "fullName": "testcases.test_AddClassification.TestAddClassification#test_2_TypeAddClass", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_AddClassification"}, {"name": "subSuite", "value": "TestAddClassification"}, {"name": "host", "value": "DESKTOP-S2QKEKE"}, {"name": "thread", "value": "8844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_AddClassification"}]}